SimCalorimetry/HGCalSimProducers/interface/HGCDigitizerBase.h

```diff
@@ -22,7 +22,7 @@
 #include "Geometry/HcalTowerAlgo/interface/HcalGeometry.h"
 
 namespace hgc = hgc_digi;
-
+using namespace std;
```

Comment:
@adas1994 , why you need this? better not to do in header file if this is not strongly needed.